Source: Mike Kazantsev (MK_FG on IRC)
Upstream: https://sourceforge.net/tracker/?func=detail&aid=3183683&group_id=10350&atid=310350
Reason:
  python-xlib-0.15rc1 has "print" in ext.randr.init method that's completely uncalled for
  and was probably just forgotten there.
  As a result, it prints an extra line each time the module is initialized (and that
  actually happens on Display class instantiation).

diff --git a/Xlib/ext/randr.py b/Xlib/ext/randr.py
index 7207bd1..19e7d0e 100644
--- a/Xlib/ext/randr.py
+++ b/Xlib/ext/randr.py
@@ -1139,8 +1139,6 @@ class OutputPropertyNotify(rq.Event):
 # Initialization #
 
 def init(disp, info):
-    print info.__class__
-
     disp.extension_add_method('display', 'xrandr_query_version', query_version)
     disp.extension_add_method('window', 'xrandr_select_input', select_input)
     disp.extension_add_method('window', 'xrandr_get_screen_info', get_screen_info)
